cg/meta/demultiplex/housekeeper_storage_functions.py

```diff
@@ -72,11 +69,7 @@ def add_sample_fastq_files_to_housekeeper(
     flow_cell: FlowCellDirectoryData, hk_api: HousekeeperAPI, store: Store
 ) -> None:
     """Add sample fastq files from flow cell to Housekeeper."""
-    sample_internal_ids: List[str] = get_sample_internal_ids_from_sample_sheet(
-        sample_sheet_path=flow_cell.get_sample_sheet_path_hk(),
-        flow_cell_sample_type=flow_cell.sample_type,
-    )
-
+    sample_internal_ids: List[str] = flow_cell.sample_sheet.get_sample_ids()
```

Comment:
```suggestion
    sample_internal_ids: Set[str] = flow_cell.sample_sheet.get_sample_ids()
```
The sample sheet has duplicated sample internal ids, might be safer to get a set.

Reply:
The function actually returns a list of unique ids already, maybe the function should be renamed to `get_unique_sample_ids` or something

Reply:
Updated the docstring of `get_sample_ids` to clarify it is a list of unique ids.